src/hb-ft.cc

```diff
@@ -120,6 +120,9 @@ _hb_ft_font_destroy (void *data)
 {
   hb_ft_font_t *ft_font = (hb_ft_font_t *) data;
 
+  if (unlikely (ft_font == nullptr))
```

Comment:
we recently had `ft_font == nullptr` to `!ft_font` like cleanup

Reply:
That's an easy change. I'll fix.